batch/batch/worker.py

```diff
@@ -757,10 +786,17 @@ def __init__(self):
                 log.info('cleaned up app runner')
 
     async def post_job_complete_1(self, job, run_duration):
+        status = await job.status()
         body = {
-            'status': await job.status()
+            'status': status
```

Comment:
Wait, this is sending the full status back to the driver?  Can we just send the database status?  That will save decoding, parsing, etc. of the status.